cfstore/plugins/posix.py

```diff
@@ -148,11 +149,22 @@ def aggregation_files_to_collection(self, aggfile,collection):
             domain = v.domain._one_line_description()
             size = v.size
             #Maybe use shape?
-            var = db.Variable(identity=identity,standard_name=name, long_name=long_name, cfdm_size=size, cfdm_domain=domain, _proxied={})
+            
+            managed_properties={}
             for k,p in properties.items():
                 if k not in ['standard_name','long_name']:
-                    var[k] = manage_types(p)
-            var.save()
+                    managed_properties[k] = manage_types(p)
+            if 'frequency' in managed_properties.keys():
+                if managed_properties['frequency'] == cf.D:
+                    managed_properties['frequency']= "Daily"
+                if managed_properties['frequency'] == cf.M:
+                    managed_properties['frequency']= "Monthly"
+                if managed_properties['frequency'] == cf.Y:
+                    managed_properties['frequency']= "Yearly"
```

Comment:
I don't follow this (although I'm sure it's fine - I can't see where the dictionary values are getting assigned to cf package functions (like `cf.Y`). `def manage_types()` doesn't seem to do it. What am I missing?

Edit - Oh hang on, should it be 
```python
                 if frequency == "Daily":
                    managed_properties["frequency"] = cf.D
```
???

```suggestion
            frequency = managed_properties.get("frequency")
            if frequency is not None:
                if frequency == cf.D:
                    managed_properties["frequency"] = "Daily"
                if frequency == cf.M:
                    managed_properties["frequency"] = "Monthly"
                if frequency == cf.Y:
                    managed_properties["frequency"] = "Yearly"
```